doc/user/nebula_cluster_helm_guide.md

```diff
@@ -75,7 +75,7 @@ The following table lists is the configurable parameters of the chart and their
 | Parameter | Description | Default |
 |:---------|:-----------|:-------|
 | `nameOverride` | Override the name of the chart | `nil` |
-| `nebula.version` | Nebula version | `v2.0.1` |
+| `nebula.version` | Nebula version | `v2.6.1` |
```

Comment:
Those now added config in values is not contains.

Reply:
I got it